tee/nix/flake.nix

```diff
@@ -37,6 +37,7 @@
       packages = {
         all = pkgs.linkFarm "all" [
           { name = "rewardTEE"; path = packages.rewardTEE; }
```

Comment:
What do we do with the other set of creates? If we don't need them, then we can just use them as baseline, instead of spawning three more crates

Reply:
Scope of https://cardinal-cryptography.atlassian.net/browse/BAC-186